app/domain/authentication/authn_k8s/inject_client_cert.rb

```diff
@@ -71,7 +72,7 @@ def validate
       def install_signed_cert
         pod_namespace = spiffe_id.namespace
         pod_name = spiffe_id.name
-        cert_file_path = "/etc/conjur/ssl/client.pem"
+        cert_file_path = @cert_path.nil? || @cert_path.empty? ? DEFAULT_CERT_PATH : @cert_installation_path
```

Comment:
this logic should probably be in its own method and can be unit tested

you can see how we do this for the `host_id_prefix` (which can also be set in a header) here:
https://github.com/cyberark/conjur/blob/73c197037a3bce4122d9ae97654344e5e265e470/app/domain/authentication/authn_k8s/inject_client_cert.rb#L57-L59

maybe we can add a unit test for the logic here: https://github.com/cyberark/conjur/blob/master/spec/app/domain/authentication/authn_k8s/inject_client_cert_spec.rb?